apps/laboratory/tests/shared/pages/ModalPage.ts

```diff
@@ -236,7 +241,7 @@ export class ModalPage {
     if (qrCodeFlowType === 'immediate-connect' || qrCodeFlowType === 'immediate') {
       uri = await page.getImmidiateConnectUri(undefined, qrCodeFlowType === 'immediate-connect')
     } else {
-      uri = await page.getConnectUri()
+      uri = await page.getConnectUri(undefined, modalOpen)
```

Comment:
nit: can we change it so we don't have to pass in an `undefined` as first param?